boms/src/main/resources/licenses-transform.xsl

```diff
@@ -0,0 +1,204 @@
+          <xsl:with-param name="url" select="'http://asm.ow2.org/license.html'"/>
+        </xsl:call-template>
+      </xsl:when>
+      <!-- antlr -->
+      <xsl:when test="ancestor::dependency/groupId/text() = 'antlr'">
+        <xsl:call-template name="license">
+          <xsl:with-param name="name" select="$bsd_3_name"/>
+          <xsl:with-param name="url" select="$bsd_3_url"/>
+        </xsl:call-template>
+      </xsl:when>
+      <!-- relaxngDatatype -->
+      <xsl:when test="ancestor::dependency/groupId/text() = 'com.github.relaxng' and ancestor::dependency/artifactId/text() = 'relaxngDatatype'">
+        <xsl:call-template name="license">
+          <xsl:with-param name="name" select="$bsd_3_name"/>
+          <xsl:with-param name="url" select="$bsd_3_url"/>
+        </xsl:call-template>
+      </xsl:when>
+      <!-- dom4j -->
+      <xsl:when test="ancestor::dependency/groupId/text() = 'dom4j' and ancestor::dependency/artifactId/text() = 'dom4j'">
+        <xsl:call-template name="license">
+          <xsl:with-param name="name" select="'The Dom4j License'"/>
+          <xsl:with-param name="url" select="'https://raw.githubusercontent.com/dom4j/dom4j/master/LICENSE'"/>
+        </xsl:call-template>
+      </xsl:when>
+      <!-- openjdk-orb -->
+      <xsl:when test="ancestor::dependency/groupId/text() = 'org.jboss.openjdk-orb' and ancestor::dependency/artifactId/text() = 'openjdk-orb'">
+        <xsl:call-template name="license">
+          <xsl:with-param name="name" select="'GNU General Public License, Version 2 with the Classpath Exception'"/>
+          <xsl:with-param name="url" select="'http://repository.jboss.org/licenses/gpl-2.0-ce.txt'"/>
+        </xsl:call-template>
+      </xsl:when>
+      <!-- jaxen -->
+      <xsl:when test="ancestor::dependency/groupId/text() = 'jaxen' and ancestor::dependency/artifactId/text() = 'jaxen'">
+        <xsl:call-template name="license">
+          <xsl:with-param name="name" select="'The Jaxen License'"/>
+          <xsl:with-param name="url" select="'http://www.jaxen.org/license.html'"/>
+        </xsl:call-template>
+      </xsl:when>
+
+      <!-- If nothing matches, leave original values -->
+      <xsl:otherwise>
+        <xsl:call-template name="license">
+          <xsl:with-param name="name" select="name/text()"/>
+          <xsl:with-param name="url" select="url/text()"/>
+          <xsl:with-param name="origin" select="'Original'"/>
+        </xsl:call-template>
+      </xsl:otherwise>
+    </xsl:choose>
+  </xsl:template>
+
+  <xsl:template name="license">
+    <xsl:param name="name"/>
+    <xsl:param name="url"/>
+    <xsl:param name="origin" select="'Fixed'"/>
+    <license>
+      <!-- For debug purposes only -->
+      <!-- xsl:comment><xsl:value-of select="$origin"/></xsl:comment-->
+      <name><xsl:value-of select="$name"/></name>
+      <url><xsl:value-of select="$url"/></url>
+    </license>
+  </xsl:template>
+
+</xsl:stylesheet>
```

Comment:
I fear we'll soon have a gazillion copies of this file, each with their own changes. We should coordinate around a single source of truth that would be just released to Maven Central. Just a comment, not a request for changing anything in this PR.